### ISSUE TITLE
License ?

### ISSUE THREAD

**Issue by suprafun** [Angus-Fan/TurnBasedStrategyGame#1]
Hello, under what license is this project released under ? I would like to study it to learn Unity. Thank-you.

**Comment by Angus-Fan** [comment]
Hey there, I believe I used the Individual Personal License. This is because I'm mainly just programming for my own projects and learning things on my own time. If you want to learn unity I would recommend getting this license as its free.

**Comment by suprafun** [comment]
Sorry to clarify, I meant your project (the source code from your very nice Youtube tutorial). I just want to read it closely while following along with your Youtube tutorial video. Thanks.

**Comment by Angus-Fan** [comment]
I believe my tutorial isn't very friendly for following step by step. I am mostly just explaining how I did certain aspects of the game. I believe that the majority of algorithms that I talk about are done in the tileMapScript.cs file found on the code section of this project. If you want to open the project itself on unity download the projectFiles.zip file from the code section and then open it with Unity.

**Comment by Hanulinen** [comment]
Hi, I want to ask, as I don't think you clarify this: AM I allowed to take this code and build my own (commercial) project on top of it? I think this is what the previous user was asking? Do you want to be credited in my hypothetical game?

**Comment by Angus-Fan** [comment]
Hey sorry I didn't see this email until now. Go ahead and use whatever you
want from my Project. If you can give me a shoutout that would be great,
but not mandatory.
Glad that you find it beneficial.

Thanks,
Angus.

**Comment by Hanulinen** [comment]
Thank you! We will credit you in the game as Angus Fan. Is that okay? Your project is a great help! Thank you for making it :)

**Comment by Angus-Fan** [comment]
Yep that's fine, do keep in mind a portion of the code that I used was modelled after the tutorials found on **Quill18Creates's** YouTube channel. (mainly the pathfinding algorithm A*)
 Playlist here: https://www.youtube.com/watch?v=kYeTW2Zr8NA&list=PLbghT7MmckI55gwJLrDz0UtNfo9oC0K1Q&ab_channel=quill18creates .
